pkg/plugin/examples/python/pyraw.yml

```diff
@@ -0,0 +1,29 @@
+# example plugin config
+name: Hawwwwt Tagger (Raw Python edition)
+description: Python Hawwwwt tagging utility (using raw interface).
+version: 1.0
+url: http://www.github.com/stashapp/stash
+exec:
+  - python
+  - {pluginDir}/pyplugin.py
```

Comment:
i had to change that to load the yml without errors
`  - "{pluginDir}/pyplugin.py`